### DISCUSSION TITLE
Bug Report: Ver. 1.4.2 isn't exporting!

### DISCUSSION THREAD

**Discussion by AubreyTheSilly** [KadeArchive/Kade-Engine#718] [Q&A] [answered]
I export it, but then it gives me this:
"error: linker command failed with exit code 1 (use -v to see invocation)"
Please help with this. The other versions work fine, but i want to use the latest version.

**Comment by puyoxyz** [comment]
1.5 is now the latest version so like uh try that

**Comment by ghost** [comment]
Hi - Please try version 1.5.1 and check if the issue persists.
If you still have this issue, add "-v" to the end of the build command.

**Comment by AubreyTheSilly** [comment]
OK it works now thx

Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________

**Comment by AwesomeWithRex** [comment, marked as answer]
ok the problem here is on titlescreen.hx if you see the lines saying:
#if polymod
polymod.Polymod.init({modRoot: "mods", dirs: ['introMod']});
#end
delete all of the 3 lines and you should test it now and it should open :)
the reason to delete these is its useless and is what makes your game crash.

**Reply by AubreyTheSilly** [reply]
Hey, I thought this bug closed a long time ago! But, thanks! I've really wanted to use Kade Engine again. Ever since, I've been using Psych Engine.

**Reply by AwesomeWithRex** [reply]
No problem dude!
